### PR TITLE
Change to new repo name.

### PR DESCRIPTION
**Pull request by scottdangelo**
`conversation` -> `assitant`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,12 +67,12 @@ When the reader has completed this Code Pattern, they will understand how to:
 
 ### 1. Clone the repo
 
-Clone the `alexa-skill-watson-conversation` repo locally and `cd` to the local repo
+Clone the `alexa-skill-watson-assitant` repo locally and `cd` to the local repo
```
Comment (markstur) [inline README.md]: typo

File: README.md [hunk 2]
```diff
@@ -67,12 +67,12 @@ When the reader has completed this Code Pattern, they will understand how to:
...
 (for commands in later steps). In a terminal, run:
 
 ```bash
-git clone https://github.com/IBM/alexa-skill-watson-conversation
-cd alexa-skill-watson-conversation
+git clone https://github.com/IBM/alexa-skill-watson-assitant
```
Comment (markstur) [inline README.md]: yeah search replace all "assitant" use

File: README.md [hunk 3]
```diff
@@ -93,7 +93,7 @@ your Assistant service and import your workspace. The service will be named
 
 Create the service by following this link and hitting `Create`:
 
-* [**Watson Assistant**](https://cloud.ibm.com/catalog/services/conversation)
+* [**Watson Assistant**](https://cloud.ibm.com/catalog/services/assitant)
```
Comment (markstur) [inline README.md]: I didn't think this link changed yet, but you are correct once the typo is fixed.

File: README.md (hunk 1 above)
Comment (scottdangelo) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (scottdangelo) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (scottdangelo) [reply README.md]: fixed

**Pull request merged**